src/meshes/p4est_mesh.jl

```diff
@@ -54,10 +70,12 @@ end
 
 
 @inline Base.ndims(::P4estMesh{NDIMS}) where NDIMS = NDIMS
-@inline Base.real(::P4estMesh{NDIMS, RealT}) where {NDIMS, RealT} = RealT
+@inline Base.real(::P4estMesh{NDIMS, IsParallel, RealT}) where {NDIMS, IsParallel, RealT} = RealT
 
 @inline ntrees(mesh::P4estMesh) = mesh.p4est.trees.elem_count
-@inline ncells(mesh::P4estMesh) = mesh.p4est.global_num_quadrants
+@inline ncells(mesh::SerialP4estMesh) = mesh.p4est.global_num_quadrants
```

Comment:
It is the same. If I recall correctly, I did it this way because it explicitly shows the difference between the serial and parallel case instead of requiring the reader to recall that local == global in the serial but local != global in the parallel case and I thought it might be easier to understand this way.

Reply:
Hah, I thought about it the other way: If both query the local, it is easier because the reader knows that local==global in the serial case, and they do not have to wonder why the global count is explicitly requested for the serial mesh.

I slightly favor having just one implementation for this function (i.e., local count for general `P4estMesh`es), but I'll leave it up to you.

Reply:
I guess it depends how much experience the reader has. But imo the `P4estMesh` isn't really intended or well-suited for beginners anyway, so I unified both methods in fd5b1f7.